### DISCUSSION TITLE
truffle migrate error: Unable to update lock within the stale threshold

### DISCUSSION THREAD

**Discussion by hadoanngoc** [ConsenSys-archive/truffle#5327] [General Q&A]
I have asked this question in OpenZeppelin forum but not found the solution yet. Hope someone here can help

So I'm running "truffle migrate --network rinkeby" to migrate contracts to the rinkeby network using infura provider. I only suceeded once. After that, (cleared cache, create new project from scratch...) I always get this error

```
   Deploying 'ComingLock'
   ----------------------
   > transaction hash:    0x884383ee2645aaee5d749ec8f62596831954d31049ba45b32c78942bda2eb8fb
   ⠸ Blocks: 0            Seconds: 0
/.../node_modules/graceful-fs/polyfills.js:306
        if (cb) cb.apply(this, arguments)
                   ^
Error: Unable to update lock within the stale threshold
...
```

I have followed the suggestion to remove the *.lock file/folder in .openzeppelin folder but still got the error. Also tried to comment out line 306 in polyfills.js file as the error shows, but it is stuck at the deployement. The migration is very simple 

```
// migrations/2_deploy.js
const { deployProxy } = require('@openzeppelin/truffle-upgrades');
const thelock = artifacts.require('ComingLock');

module.exports = async function (deployer) {
  await deployProxy(thelock, [], { deployer });
};
```

Anyone knows the solution?

**Comment by hadoanngoc** [comment]
Commenting out line 131-139 in /node_modules/proper-lockfile/lockfile.js (as shown in the error) did help. But not sure if it has consequences

**Comment by eggplantzzz** [comment]
Hey @hadoanngoc! I'm not too familiar with the stuff you are using from openzeppelin, is there any chance you can link to a tiny example on GitHub that will allow me to reproduce your problem? I'll try and reproduce it and see if I get any ideas about what might be wrong.
